projects/cli-files/go-cat/cmd/root.go

```diff
@@ -2,34 +2,29 @@ package cmd
 
 import (
 	"bufio"
+	"embed"
 	"flag"
 	"fmt"
 	"os"
 )
 
+//go:embed help.txt
+var helpFile embed.FS
```

Comment:
Ooh, interesting use of embed, this is cool!

One of the interesting things about embed is you can embed as different types - if you declared this variable to be a `string` instead of an `embed.FS`, you could just use it as a string rather than needing to call `.ReadFile` and handle a potential error.